lib/diagnostic.rb

```diff
@@ -13,28 +13,32 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(:+)
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    x = false
+    numbers.each {|num| num.odd? ? x = true : x }
+    return x
```

Comment:
This will work, but there's a more elegant approach: `numbers.any? { |num| num.odd? }`
This can also be shortened as `numbers.any?(&:odd?)` / `numbers.any?(:odd)`